chainer/links/connection/gru.py

```diff
@@ -53,7 +53,7 @@ class StatelessGRU(GRUBase):
     where :math:`\\sigma` is the sigmoid function, and :math:`\\odot` is the
     element-wise product.
 
-    :class:`~chainer.links.GRU` does not hold the value of
+    :class:`~chainer.links.StatelessGRU` does not hold the value of
     hidden vector :math:`h`. So this is *stateless*.
```

Comment:
> So this is *stateless*

This sentence is now somewhat odd as users can infer it from the name. So I think something like "As the name indicates, StatelessGRU is *stateless*, meaning that it does not hold ..." is better.

Reply:
Thank you! Fixed.